src/components/containers/fieldset/Fieldset.js

```diff
@@ -40,7 +40,7 @@ function Fieldset({ additionalLegendClasses, legendText, children, theme }) {
 
 Fieldset.propTypes = {
   /** Determine whether or not to add a legend and what text to display  */
-  legendText: PropTypes.string,
+  legendText: PropTypes.object,
```

Comment:
This seems a little odd calling it "Text" when it's now an object. What do you think about creating a subcomponent for <Fieldset.Legend>?